src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -1156,6 +1156,18 @@ void CascadiaSettings::ExportFile(winrt::hstring path, winrt::hstring content)
 
 void CascadiaSettings::_validateThemeExists()
 {
+    if (_globals->Themes().Size() == 0)
+    {
+        // GH #13474
+        // We didn't even load the default themes. This should only be possible
+        // in the tests that don't even pass in defaults.json. No matter. Create

```

Comment:
Comments that say "this never happens" _do not age well_; reconsider!

(also: linking it to a github ID isn't meaningful in this case. we just should always make sure there's a theme, regardless of a github issue. I generally hold that if we take this to its logical extreme, every line of code will have its own github issue . . . and then it's no better than `git blame` but in many ways worse)